include/exadg/structure/material/library/st_venant_kirchhoff.cpp

```diff
@@ -155,40 +154,36 @@ StVenantKirchhoff<dim, Number>::second_piola_kirchhoff_stress_symmetrize(
     f2 = f2_coefficients.get_coefficient_cell(cell, q);
   }
 
-  if(dim == 3)
+  // Since we return a `symmetric_tensor`, we only need to set symmetric entries once.
+  if constexpr(dim == 3)
   {
     S[0][0] = f0 * strain[0][0] + f1 * (strain[1][1] + strain[2][2]);
     S[1][1] = f0 * strain[1][1] + f1 * (strain[0][0] + strain[2][2]);
     S[2][2] = f0 * strain[2][2] + f1 * (strain[0][0] + strain[1][1]);
     S[0][1] = f2 * (strain[0][1] + strain[1][0]);
     S[1][2] = f2 * (strain[1][2] + strain[2][1]);
     S[0][2] = f2 * (strain[0][2] + strain[2][0]);
-    S[1][0] = S[0][1];
```

Comment:
here we explicitly see that we can save some computations, but it is rather in the operations with the symmetric tensors that we automatically profit whenever `dealii::SymmetricTensor`s are can be used.